src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -204,12 +206,11 @@ public void onGuildVoiceLeave(GuildVoiceLeaveEvent event) {
                 UpdateCommand.update(true, null);
             }
         } else {
-            if (event.getMember().getUser().equals(event.getJDA().getSelfUser())) {
-                return;
-            }
-            if (event.getChannelLeft().getMembers().size() < 2 || event.getChannelLeft().getMembers()
-                    .stream().filter(m -> m.getUser().isBot()).count() == event.getChannelLeft().getMembers().size()) {
-                event.getChannelLeft().getGuild().getAudioManager().closeAudioConnection();
+            if (event.getChannelLeft().getMembers().contains(event.getGuild().getSelfMember())) {
+                if (event.getChannelLeft().getMembers().size() < 2 || event.getChannelLeft().getMembers()
```

Comment:
Make these two ifs together with an && instead of apart. It's literally the only thing in the if, it will be ugly but it's what we should do.